### PR TITLE
added animated button, Marks

### PR DESCRIPTION
**Pull request by AnastasiaOsinska**

### CODE REVIEW

File: FacebookTraineeProject/ViewControllers/WelcomePageViewController/WelcomePageViewController.swift [hunk 1]
```diff
@@ -8,17 +8,34 @@
...
+    static let loginButtonRadius: CGFloat = 8
+    static let offsetPoint: CGFloat = 0
+}
+
+struct Animation {
+    static let durationDelay: Double = 1
+    static let damping: CGFloat = 0.47
+    static let velocity: CGFloat = 1
+    static let loginButtonAltha: CGFloat = 1
+}
```
Comment (ShurinVitaliy) [inline FacebookTraineeProject/ViewControllers/WelcomePageViewController/WelcomePageViewController.swift:23]: you have to put it in the controller
also rename it
make private struct Constants

File: FacebookTraineeProject/ViewControllers/WelcomePageViewController/WelcomePageViewController.swift [hunk 2]
```diff
@@ -37,16 +54,18 @@ class WelcomePageViewController: UIViewController {
...
     
+    // MARK: - Methods
+    
     func animation(){
-        let offset = CGPoint(x: 0, y: -loginButton.frame.maxY)
-        let x: CGFloat = 0, y: CGFloat = 0
+        let offset = CGPoint(x: Size.offsetPoint, y: -loginButton.frame.maxY)
+        let x: CGFloat = Size.offsetPoint, y: CGFloat = Size.offsetPoint
         loginButton.transform = CGAffineTransform(translationX: offset.x + x, y: offset.y + y)
         loginButton.isHidden = false
```
Comment (ShurinVitaliy) [inline FacebookTraineeProject/ViewControllers/WelcomePageViewController/WelcomePageViewController.swift:63]: why is this here?

Comment (AnastasiaOsinska) [reply FacebookTraineeProject/ViewControllers/WelcomePageViewController/WelcomePageViewController.swift:63]: who knows))
